code/game/objects/items/weapons/handcuffs.dm

```diff
@@ -65,7 +65,7 @@
 /obj/item/weapon/restraints/handcuffs/cable
 	name = "cable restraints"
 	desc = "Looks like some cables tied together. Could be used to tie something up."
-	icon_state = "cuff_red"
+	icon_state = "cuff_white"

```

Comment:
It works fine.
![https://i.imgur.com/Cdl7Rx6.png](https://i.imgur.com/Cdl7Rx6.png)